tinygrad/ops.py

```diff
@@ -224,8 +225,7 @@ def backward(ctx, grad_output):
 
     # needs to be optimized
     gdx = np.zeros((bs,ctx.groups,cin,OY,OX), dtype=tx.dtype)
-    for Y in range(grad_output.shape[2]):
-      for X in range(grad_output.shape[3]):
+    for Y, X in itertools.product(range(grad_output.shape[2]),range(grad_output.shape[3])):
```

Comment:
Think the left side is more readable

Reply:
i was writing this in context of reduction of lines

Reply:
i can remove it though :)

Reply:
sure!